agenta-backend/agenta_backend/routers/container_router.py

```diff
@@ -26,16 +20,24 @@
 else:
     from agenta_backend.services import container_manager
 
-import logging
+from agenta_backend.models.api.api_models import (
+    URI,
+    Image,
+    RestartAppContainer,
+    Template,
+)
+from agenta_backend.services import db_manager
+from agenta_backend.utils.common import APIRouter
 
-logger = logging.getLogger(__name__)
 
+logger = logging.getLogger(__name__)
 logger.setLevel(logging.DEBUG)
 
 router = APIRouter()
 
 
 # TODO: We need to improve this to use the introduced abstraction to also use start and stop service
+# * Edit: someone remind me (abram) to work on this.
```

Comment:
@aybruhm work on this!

Reply:
Thank you for the reminder. I'll create an issue for it later today.